src/pages/community.astro

```diff
@@ -0,0 +1,303 @@
+          FCP cancelled
+          <svg class="absolute w-8 h-8 -left-[calc(2rem_+_2px)] top-0" width="30" height="30" viewBox="0 0 30 30" fill="none" xmlns="http://www.w3.org/2000/svg">
+            <path class="stroke-2 stroke-gray-300" d="M30 29.5C26.126 29.5 22.2899 28.737 18.7108 27.2544C15.1317 25.7719 11.8797 23.599 9.14035 20.8596C6.40102 18.1203 4.22807 14.8683 2.74555 11.2892C1.26304 7.71005 0.5 3.87399 0.5 -2.57897e-06"/>
+          </svg>
+          <hr class="absolute border border-gray-300 h-24 w-0 -left-[calc(2rem_+_2px)] -top-[calc(2rem_+_2px)]"/>
+        </div>
+        <div class="border-2 border-gray-300 bg-gray-100 rounded-xl px-4 py-4 text-lg w-64 rfc-box-x relative ml-12">
+          <svg class="inline relative stroke-base-content -top-0.5 rfc-box-x" xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-file-x-2"><path d="M4 22h14a2 2 0 0 0 2-2V7.5L14.5 2H6a2 2 0 0 0-2 2v4"/><path d="M14 2v6h6"/><path d="m3 12.5 5 5"/><path d="m8 12.5-5 5"/></svg>
+          Submission rejected
+          <svg class="absolute w-8 h-8 -left-[calc(2rem_+_2px)] top-0" width="30" height="30" viewBox="0 0 30 30" fill="none" xmlns="http://www.w3.org/2000/svg">
+            <path class="stroke-2 stroke-gray-300" d="M30 29.5C26.126 29.5 22.2899 28.737 18.7108 27.2544C15.1317 25.7719 11.8797 23.599 9.14035 20.8596C6.40102 18.1203 4.22807 14.8683 2.74555 11.2892C1.26304 7.71005 0.5 3.87399 0.5 -2.57897e-06"/>
+          </svg>
+          <hr class="absolute border border-gray-300 h-24 w-0 -left-[calc(2rem_+_2px)] -top-[calc(2rem_+_2px)]"/>
+        </div>
+        <div class="border-2 border-gray-300 bg-gray-100 rounded-xl px-4 py-4 text-lg w-64 rfc-box-4 relative ml-12">
+          <svg class="inline relative stroke-base-content -top-0.5" xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-file-check-2"><path d="M4 22h14a2 2 0 0 0 2-2V7.5L14.5 2H6a2 2 0 0 0-2 2v4"/><polyline points="14 2 14 8 20 8"/><path d="m3 15 2 2 4-4"/></svg>
+          Submission accepted
+          <svg class="absolute w-8 h-8 -left-[calc(2rem_+_2px)] top-0" width="30" height="30" viewBox="0 0 30 30" fill="none" xmlns="http://www.w3.org/2000/svg">
+            <path class="stroke-2 stroke-gray-300" d="M30 29.5C26.126 29.5 22.2899 28.737 18.7108 27.2544C15.1317 25.7719 11.8797 23.599 9.14035 20.8596C6.40102 18.1203 4.22807 14.8683 2.74555 11.2892C1.26304 7.71005 0.5 3.87399 0.5 -2.57897e-06"/>
+          </svg>
+          <hr class="absolute border border-gray-300 h-[calc(2rem_+_2px)] w-0 -left-[calc(2rem_+_2px)] -top-[calc(2rem_+_2px)]"/>
+        </div>
+      </div>
+    </div>
```

Comment:
> the animation lacks some steps from the original graph which strips some context away which should be there to completely understand the review process.

Is this okay?

![image](https://github.com/NixOS/nixos-homepage/assets/97523625/4a6e87a5-c966-414b-994e-43a765b130c6)

Or should I add more missing nodes (`Refine submission`, `New submission` and `Substantial new arguments`)? I'm asking, because it would require changes to graph introduced in this PR. Old graph for reference:

![image](https://github.com/NixOS/nixos-homepage/assets/97523625/46cef168-6169-47b3-b0fc-20b0105a4403)

Reply:
The difference between your graph and the one currently used is, that the currently used one at a glance and without an animation shows which paths the RFC process can take and can also be percieved by people who, for example, have a reduced motion preference set in their browser, since they likely wont get to see the animation and cannot percieve the whole context this image should offer. Apart from that, the animation and the graph you submitted lack the steps of "Refine..." and "New...", which make a big difference in how submissiosn are handeled between processes. 

So tl:dr; yes, please include all branches into your graph since they provide a version of the graph that is percieveable by every visitor. Animations are fine, but they should just enhance the context and not communicate crucial context of a graph.